inspire/base/static/less/format/brief-results.less

```diff
@@ -84,4 +84,15 @@ hr {
 		border-right: 0px;
 	}
 
-}
\ No newline at end of file
+}
+
+@media screen and (min-width: 768px) {
```

Comment:
can you reindent this file to have 4 spaces (like for example https://github.com/ioannistsanaktsidis/inspire-next/blob/latex-eu-us/inspire/base/static/less/feedback/modal.less)